content/influxdb/v2.0/query-data/common-queries/find_unique_values_for_column.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Find unique values for a column
```

Comment:
This basically does the exact same thing as the above example, but doesn't count them. Do we think we need both?

Reply:
I think you can combine them into one doc. Have one section that shows how to get unique values and another section that shows how to count those values.